extra/man/alacritty.5.scd

```diff
@@ -0,0 +1,800 @@
+
+		If the bright foreground color is not set, it is automatically
+		calculated based on the foreground color.
+
+		Default: _"#828482"_
+
+	*bright_foreground* <string>
+
+		If the bright foreground color is not set, or
+		_draw\_bold\_text\_with\_bright\_colors_ is _false_, the normal
+		foreground color will be used.
+
+		Default: _"#eaeaea"_
+
+*cursor* { text = <string>, cursor = <string> }
+
+	Colors which should be used to draw the terminal cursor.
+
+	Allowed values are hexadecimal colors like _#ff00ff_, or
+	_CellForeground_/_CellBackground_, which references the affected cell.
+
+	Default: _{ text = "CellBackground", cursor: "CellForeground" }_
+
+*vi_mode_cursor* { text = <string>, cursor = <string> }
+
+	Colors for the cursor when the vi mode is active.
+
+	Allowed values are hexadecimal colors like _#ff00ff_, or
+	_CellForeground_/_CellBackground_, which references the affected cell.
+
+	Default: _{ text = "CellBackground", cursor: "CellForeground" }_
+
+*search*
+
+	Colors used for the search bar and match highlighting.
+
+	Allowed values are hexadecimal colors like _#ff00ff_, or
+	_CellForeground_/_CellBackground_, which references the affected cell.
+
+	*matches* { foreground = <string>, background = <string> }
+
+		Default: _{ foreground = "#000000", background: "#ffffff" }_
+
+	*focused_match* { foreground = <string>, background = <string> }
+
+		Default: _{ foreground = "#ffffff", background: "#000000" }_
+
+*hints*
+
+	*start* { foreground = <string>, background = <string> }
+
+		First character in the hint label
+
+		Allowed values are hexadecimal colors like _#ff00ff_, or
+		_CellForeground_/_CellBackground_, which references the affected cell.
+
+		Default: _{ foreground = "#1d1f21", background = "#e9ff5e" }_
+
+	*end* { foreground = <string>, background = <string> }
+
+		All characters after the first one in the hint label
+
+		Allowed values are hexadecimal colors like _#ff00ff_, or
+		_CellForeground_/_CellBackground_, which references the affected cell.
+
+		Default: _{ foreground = "#e9ff5e", background = "#1d1f21" }_
+
+*line_indicator* { foreground = <string>, background = <string> }
+
+	Color used for the indicator displaying the position in history during
+	search and vi mode.
+
+	Setting this to _"none"_ will use the opposing primary color.
+
+	Default: _{ foreground = "none", background = "none" }_
+
+*footer_bar* { foreground = <string>, background = <string> }
+
+	Color used for the footer bar on the bottom, used by search regex input,
+	hyperlink URI preview, etc.
+
+	Default: _{ foreground = "#c5c8c6", background = "#1d1f21" }_
+
+*selection* { text = <string>, background = <string> }
+
+	Colors used for drawing selections.
+
+	Allowed values are hexadecimal colors like _#ff00ff_, or
+	_CellForeground_/_CellBackground_, which references the affected cell.
+
+	Default: _{ text = "CellBackground", background = "CellForeground" }_
+
+*normal*
+
+	*black* <string>
+		Default: _"#1d1f21"_
```

Comment:
I see that you want me to port this as well...

Reply:
:)